src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Order/TreeItemOrderPropertyProvider.cs

```diff
@@ -93,6 +143,10 @@ private void ComputeIndices()
                     // (this will typically be hidden items visible on "Show All Files")
                     propertyValues2.DisplayOrder = int.MaxValue;
                 }
+                else
+                {
+                    propertyValues2.DisplayOrder = 0;
```

Comment:
These are nodes you don't know - why are setting the display order? Won't this affect the display order of say "Properties"?

Reply:
DisplayOrder is always zero for all items in the beginning, including "Properties". The only time DisplayOrder ever changes is within this provider. Setting display order to zero at this point ensures that everything else stays at zero.

Reply:
> The only time DisplayOrder ever changes is within this provider
This may be true right now - but CPS is an extensible system, other teams add nodes to the solution explorer, such as Service Capabilities. We should avoid setting DisplayOrder for nodes we don't own or know about. What issue is this fixing?

Reply:
That is a good point. I guess it's not right to assume. This particular addition isn't fixing anything; so that there may be a red flag to not have this bit. :)